services/movie.py

```diff
@@ -0,0 +1,36 @@
+import init_django_orm  # noqa: F401
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    return movies.distinct()
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> Movie:
+    new_movie = Movie.objects.get_or_create(title=movie_title)[0]
```

Comment:
you didn't understand me clearly:) I meant it is better to use get_or_create if you want to check if object exists. But in this case it is redundant because this method named `create_movie`!!, not `get_or_create_movie`. delete it. and use Movie.objects.create()

Reply:
Let me explain my vision. But why not to use function "create_movie" with doube purpose? For example we have film with no relations with Genre and Actor models. And we can either create film or update relations with Genre and Actor. Why not?)
Are make additioal features in deployment withount tech specs is not a good practice? Yes?